gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -486,9 +490,19 @@ def run_translation_averaging(
             absolute_pose_priors=absolute_pose_priors,
             scale_factor=scale_factor,
         )
+        end_lm_sec = time.time()
+        lm_duration_sec = end_lm_sec - start_lm_sec
 
         # Compute the metrics.
-        ta_metrics = compute_metrics(set(w_i2Ui1_dict_inliers.keys()), i2Ui1_dict, wRi_list, wti_list, gt_wTi_list)
+        ta_metrics = compute_metrics(
+            set(w_i2Ui1_dict_inliers.keys()),
+            i2Ui1_dict,
+            wRi_list,
+            wti_list,
+            gt_wTi_list,
+            inlier_computation_duration_sec,
+            lm_duration_sec,
+        )
 
```

Comment:
compute metrics as before, simply use ta_metrics.add_metric(GtsfmMetric('my_time_sec', time)) to add the time. I dont think it needs to go into the function that does the evaluation.